pages/common/git-show-ref.md

```diff
@@ -0,0 +1,20 @@
+# git show-ref
+
+> Git command for listing references.
+> More information: <https://git-scm.com/docs/git-show-ref>.
+
+- Show all refs in the repository:
+
+`git show-ref`
+
+- Show only heads references:
+
+`git show-ref --heads`
+
+- Show only tags references:
```

Comment:
Perhaps this will read better?
```suggestion
- Show only references with an associated tag:
```

Reply:
same reasons for heads

Reply:
I see. It's just it reads oddly, is all.